apps/storybook/src/Slider.stories.tsx

```diff
@@ -20,19 +20,36 @@ export default {
   args: {
     thumbMode: 'inhibit-crossing',
     trackDisplayMode: 'auto',
+    orientation: 'horizontal',
```

Comment:
Have one story for vertical case.

Reply:
Added a story at the end called `SomeVerticalExamples`. It contains three vertical sliders as examples. Please let me know if you want me to change the contents of this story as I wasn't sure what would be the best way to demo vertical sliders

Reply:
Maybe just have one basic vertical example and name story `Vertical`

Reply:
I added one example story named `Vertical` that is just the `Basic` example with `orientation: 'vertical'`